README.md

```diff
@@ -41,16 +41,24 @@ It creates a task 'nunit' that may be configured as follows:
         // optional - defaults to FALSE and termines the behavior of the task if the nunit-console.exe program exits
         // abnormally
         ignoreFailures = false
-
+        // optional - specify whether to write test case names to the output
+        labels = 'Off|On|All'
         // Mapped NUnit-Console Command Line Options
         exclude = 'Database'
         include = 'BaseLine'
         // for NUnit v3+, use _where_ option instead of include/exclude
         framework = 'net-1.1'
-        shadowCopy = true
+        shadowCopy = true        
+        // redirect output to file
+        // additionally you can specify logFolder
+        logFileName = 'TestOutput.log'
```

Comment:
did you consider putting a single argument (`outputFile` or something) for both ? reportFolder was split because it's used in other places, not sure we need the same here and it'll be simpler.